packages/app/src/sandbox/compile.ts

```diff
@@ -657,14 +658,16 @@ async function compile({
         const htmlEntries = templateDefinition.getHTMLEntries(configurations);
         const htmlModulePath = htmlEntries.find(p => Boolean(modules[p]));
         const htmlModule = modules[htmlModulePath];
-
-        const { head, body } = getHTMLParts(
-          htmlModule && htmlModule.code
-            ? htmlModule.code
-            : template === 'vue-cli'
-            ? '<div id="app"></div>'
-            : '<div id="root"></div>'
-        );
+        let html = '<div id="root"></div>';
+        if (htmlModule && htmlModule.code) {
+          if (htmlModule.code) {
```

Comment:
I think here there should be html = htmlModule.code and in the if statement the Vue check.

Reply:
Ah, yeah, so:

```ts
let html = template === 'vue-cli' ? '<div id="app"></div>' : '<div id="root"></div>';
if (htmlModule && htmlModule.code) {
  html = htmlModule.code
}
```

Reply:
truuu, just pushed